pep-0373.txt

```diff
@@ -15,11 +15,11 @@ Abstract
 
 This document describes the development and release schedule for
 Python 2.7.  The schedule primarily concerns itself with PEP-sized
-items.  Small features may be added up to and including the first beta
-release.  Bugs may be fixed until the final release.
+items.  Small features will no longer be added.  Bugs may be fixed
+until the final release.
 
-Update
-======
+Update (April 2014)
+===================
```

Comment:
:+1: Good idea adding the date, it was a bit confusing earlier.

Reply:
Thanks!